src/app.rs

```diff
@@ -685,6 +715,11 @@ impl SsbhApp {
                     ui.close_menu();
                     self.ui_state.render_settings_open = true;
                 }
+
+                if ui.button("Hide Expressions").clicked() {
```

Comment:
This should be part of a "Meshes" menu between the "Menu" and "View" menus. `egui::menu::menu_button` is used for creating a top level menu.